Examples/Tests/TestsTests/TestsTests.swift

```diff
@@ -63,19 +63,19 @@ class TestsTests: XCTestCase {
         self.waitForExpectations(timeout: 10, handler: nil)
 
         let backgroundRealtimeExpectation = self.expectation(description: "Realtime in a Background Queue")
-        var realtime: ARTRealtime! //strong reference
+        var realtime: ARTRealtimeInternal! //strong reference
```

Comment:
Whoops, this was changed automatically but then left unfixed because it was outside the main test suite.

Anyway, CI was working because we aren't even testing this anymore. I've brought it back at edcfc4e.